rust/pjdfstest.toml

```diff
@@ -17,3 +17,17 @@ posix_fallocate = {}
 # naptime is the duration of various short sleeps.  It should be greater than
 # the timestamp granularity of the file system under test.
 naptime = 0.001
+
+# This section allows to modify the mecanism for switching users, which is required by some tests.
+[dummy_auth]
+# An entry is the name of a user and its associated group.
+# For now, the array requires 6 entries exactly.
+# Please see the book for more details.
+# entries = [
+#   "pjdfs_dummy_1",
+#   "pjdfs_dummy_2",
+#   "pjdfs_dummy_3",
+#   "pjdfs_dummy_4",
+#   "pjdfs_dummy_5",
+#   "pjdfs_dummy_6",
```

Comment:
To avoid creating too many new users, we should use `nobody` for one of the users.  We can rely on that user existing.

Reply:
Oh yes, you're right!